Block/Redirect/Redirect.php

```diff
@@ -474,11 +474,7 @@ public function getMd()
      */
     public function getTermUrl()
     {
-        if ($termUrl = $this->getPayment()->getAdditionalInformation('termUrl')) {
-            return $termUrl;
-        }
-
-        throw new AdyenException("No termUrl is provided.");
+        return $this->getUrl('adyen/transparent/redirect', ['_secure' => $this->_getRequest()->isSecure()]);
```

Comment:
We should test and use `Adyen\Payment\Helper\Data::getOrigin()` here to build this transparent endpoint's URL also for the PWA integrations that might use it

Reply:
We created a separate ticket to investigate it